bootstrap_daterangepicker/widgets.py

```diff
@@ -9,7 +9,14 @@
 from django.utils.safestring import mark_safe
 from django.utils.translation import gettext as _
 
-__all__ = ['DatePickerWidget', 'DateRangeWidget', 'DateTimeRangeWidget', 'add_month', 'common_dates']
+__all__ = [
+    'DatePickerWidget',
+    'DateTimePickerWidget',
+    'DateRangeWidget',
+    'DateTimeRangeWidget',
+    'add_month',
+    'common_dates'
```

Comment:
Do we also want to add `'common_datetimes'` here?

Reply:
Done :)